website/client/components/inventory/equipment.vue

```diff
@@ -65,12 +65,15 @@
           v-for="(label, group) in gearTypesToStrings",
           :key="group",
           :item="flatGear[activeItems[group]]",
+          :showPopover="flatGear[activeItems[group]] && flatGear[activeItems[group]].key.indexOf('_base_0') === -1",
```

Comment:
so, item wasn't available, then after some issues with the scoped context it works, just the popover-issue now also happen in side the drawer (without the scope it didn't)